esmvaltool/recipes/recipe_runoff_et.yml

```diff
@@ -0,0 +1,45 @@
+# recipe_runoff_et.yml
+---
+documentation:
+  description:
+    Recipe for plotting runoff and ET diagnostics. The diagnostics calculates water
+    balance components for different catchments and compares the results against
+    observations. Currently, the required catchment mask needs to be downloaded manually
+    at https://doi.org/10.5281/zenodo.2025776
+
+  authors:
+    - hage_st
+    - loew_al
+    - stac_to
+
+  references:
+    - duemenil00mpimr
+    - hagemann13jadvmodelearthsyst
+    - weedon14wrr
+
+  projects:
+    - embrace
+    - crescendo
+
+datasets:
+  - {dataset: MPI-ESM-LR,    project: CMIP5, exp: historical,  ensemble: r1i1p1,  start_year: 1970,  end_year: 2000}
+  - {dataset: IPSL-CM5B-LR,  project: CMIP5, exp: historical,  ensemble: r1i1p1,  start_year: 1970,  end_year: 2000}
+
+diagnostics:
+  catchment_analysis:
+    description: EMBRACE runoff and land surface evaporation diagnostic
+    variables:
+      evspsbl:
+        mip: Amon
+        field: T2Ms
+      mrro:
+        mip: Lmon
+        field: T2Ms
+      pr:
+        mip: Amon
+        field: T2Ms
+    scripts:
+      catchment_analysis:
+        script: runoff_et/catchment_analysis.py
+        catchmentmask: /home/mpim/m214103/Remote/Mistral_Work/Misc_Data/runoff_et/catchmentmask_v01.nc
```

Comment:
Having an absolute path in the recipe reduces its portability.
I would suggest setting this path to the diagnostic path (`runoff_et/catchmentmask_v01.nc`), with the caveat the the user has to download the nc file himself, as you already mentioned in the description.

Reply:
That's also fine.
On the long term we can think about adding something like a `download_path` in `config-user.yml` for such cases.